app/routes/pages/components/LandingPage.tsx

```diff
@@ -86,7 +86,7 @@ const LandingPage: PageRenderer = ({ loggedIn }) => {
             email="abakus@abakus.no"
             logo="https://raw.githubusercontent.com/webkom/lego/master/assets/abakus_hs.png"
           />
-          <h3 className={styles.title}>E-postadresser til komiteene</h3>
+          <h3 className={styles.title}>E-post-adresser til komiteene</h3>
```

Comment:
```suggestion
          <h3 className={styles.title}>E-postadresser til komiteene</h3>
```
No hyphen here https://ordbokene.no/bm/70008